test_runner/regress/test_tenant_size.py

```diff
@@ -467,16 +497,20 @@ def test_single_branch_get_tenant_size_grows(
 
     assert size_after == prev, "size after restarting pageserver should not have changed"
 
-    ps_metrics = parse_metrics(http_client.get_metrics(), "pageserver")
-    tenant_metric_filter = {
-        "tenant_id": str(tenant_id),
-    }
+    # TODO:
+    # Pageserver doesn't update prometheus metric on manual tenant size request
+    # Fix this or drop this test piece
+    #
+    # ps_metrics = parse_metrics(http_client.get_metrics(), "pageserver")
+    # tenant_metric_filter = {
+    #     "tenant_id": str(tenant_id),
+    # }
 
-    tenant_size_metric = int(
-        ps_metrics.query_one("pageserver_tenant_synthetic_size", filter=tenant_metric_filter).value
-    )
+    # tenant_size_metric = int(
+    #     ps_metrics.query_one("pageserver_tenant_synthetic_size", filter=tenant_metric_filter).value
+    # )
 
-    assert tenant_size_metric == size_after, "API size value should be equal to metric size value"
+    # assert tenant_size_metric == size_after, "API size value should be equal to metric size value"
```

Comment:
Could this be moved to consumption metrics testing?

Reply:
I am inclined to remove this at all. This code tests prometheus metric, which now simply mirrors the consumption metric.

If anything goes wrong, it will be easier to notice it in Grafana, than in python test.

Reply:
I am fine with that.